e107_themes/bootstrap3/languages/English_admin.php

```diff
@@ -18,6 +18,8 @@
 define("LAN_THEMEPREF_08", "right");
 define("LAN_THEMEPREF_09", "top");
 define("LAN_THEMEPREF_10", "bottom");
- 
+define("LAN_THEMEPREF_11", ""); // unused
+define("LAN_THEMEPREF_12", "Return to Front Panel");
+define("LAN_THEMEPREF_13", "Change Language");  
 
```

Comment:
These are not theme prefs, but part of the admin area. So, they should be included in lan_admin.php Thanks.